joint_names:
  - shoulder_yaw_joint_left
  - shoulder_pitch_joint_left
  - shoulder_roll_joint_left
  - elbow_pitch_joint_left
  - elbow_roll_joint_left
  - wrist_pitch_joint_left
  - wrist_yaw_joint_left
